pkg/reconcile/reconcile.go

```diff
@@ -0,0 +1,272 @@
+package reconcile
```

Comment:
I don't think this new package name is any better than the old one. We have 3 different controllers and all of them do the reconciliation.